src/main/java/com/example/dto/CreateBookRequestDto.java

```diff
@@ -1,14 +1,23 @@
 package com.example.dto;
 
+import com.example.validation.Isbn;
+import jakarta.validation.constraints.Min;
+import jakarta.validation.constraints.NotNull;
 import java.math.BigDecimal;
 import lombok.Data;
 
 @Data
 public class CreateBookRequestDto {
+    @NotNull
     private String title;
+    @NotNull
     private String author;
+    @NotNull
```

Comment:
not blank is more preferred as it checks string for not being null, not empty and not have only spaces
```suggestion
    @NotBlank
    private String title;
    @NotBlank
    private String author;
    @NotBlank
```

Reply:
done